index.js

```diff
@@ -39,13 +39,13 @@ module.exports = (rootDirs, { delay = 50, suppressReporting } = {}) => {
 		}
 	});
 
-	return {
-		on(name, fn) {
-			emitter.on(name, fn);
-		},
-		terminate() {
+	let wrapper = {
+		terminate: function() {
 			watcher.close();
-		}
+		};
+	};
+	return new Proxy(emitter, {
+		get: (target, prop, receiver) => wrapper[prop] || target[prop];
```

Comment:
That semicolon is a little much 👅

Reply:
Also, the semicolon above, plus a missing parenthesis below... 🙄 

Fixed in 906f1173e5cf71ff5f4794add4ad60a21fd438e3, released as v3.3.1.

Reply:
👍